### PR TITLE
add cmds and subcmds, rm flags

### PR DESCRIPTION
**Pull request by pjaudiomv**
I implemented this mostly using these suggestions https://github.com/radiusmethod/kxd/pull/14#issuecomment-1732614394

### CODE REVIEW

File: src/cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	"fmt"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "kxd",
+	Short: "kxd - switch between Kubeconfigs and contexts.",
+	Long:  "Allows for switching kubeconfig files and contexts, as well as getting the current set ones.",
+	// Should we default to kxd file switch?
```
Comment (pjaudiomv) [inline src/cmd/root.go:13]: should root just run the config switcher if `kxd` run with no flags?

**Review by dgershman — APPROVED** [review]

**Comment by dgershman** [comment]
this looks like it was very satisfiying to write.

File: src/cmd/file.go [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+	"sort"
+	"strings"
+)
+
+var fileCmd = &cobra.Command{
+	Use:     "file",
+	Short:   "Kubeconfig file command",
+	Aliases: []string{"f"},
+	Long:    "This is the default file command.",
+	//	Should this default to file switch?
```
Comment (pjaudiomv) [inline src/cmd/file.go:20]: should config switcher be run if `kxd file` ran without `kxd file switch`?

**Comment by pjaudiomv** [comment]
> this looks like it was very satisfying to write.

totes have 2 opinionated questions. I didnt care was curious what you thought, i left it to be explicit

File: src/cmd/file.go (hunk 2 above)
Comment (dgershman) [reply src/cmd/file.go:20]: i would keep it explicit for now

File: src/cmd/root.go (hunk 1 above)
Comment (dgershman) [reply src/cmd/root.go:13]: i would keep things explicit

**Comment by dgershman** [comment]
keep things explicit for now

**Comment by dgershman** [comment]
just my opinion, maybe after people use it, they might have an opinion about how to evolve it.

**Pull request merged**